core/controllers/contributor_dashboard.py

```diff
@@ -420,9 +420,10 @@ def get(self):
         content_ids = []
         try:
             content_ids = json.loads(content_ids_string)
-        except:
+        except Exception as e:
```

Comment:
In some places you use `error`, in some `exception` and in other `e`, is there any reason for that? If yes can you explain, if no can you unify the names? I think `e` should be fine.

Reply:
Made the changes.